src/BreadcrumbItem.js

```diff
@@ -7,7 +7,9 @@ import { bsClass, getClassSet, splitBsProps } from './utils/bootstrapUtils';
 
 const propTypes = {
   /**
-   * If set to true, renders `span` instead of `a`
+   * If set to true, renders `span` instead of `a` and
+   *
+   * adds `aria-current="page"` attribute
```

Comment:
The original wording here was a bit wonky maybe: "Add's a visual "active" state to a Breadcrumb Item and disabling the link.